local-cli/generator/templates/_buckconfig

```diff
@@ -0,0 +1,9 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
+
+[maven_repositories]
+  central = https://repo1.maven.org/maven2
+
+[alias]
+  android-app = //android/app:app
```

Comment:
2 things motivated me:
- `buck run android-app` looks similar to `react-native run-android` that we run from the root as well
- it is a showcase of a BUCK feature that may be not obvious

Do you think I should remove it?